frontend/www/js/omegaup/arena/arena.js

```diff
@@ -1863,8 +1863,8 @@ export class Arena {
             ' / ' +
             parseFloat(maxScore || '0').toFixed(self.digitsAfterDecimalPoint) +
             ')');
-    $('.omegaup-scoreboard tr.' + OmegaUp.username + ' td.' + alias +
-      ' .points')
+    let fixedUsername = OmegaUp.username.replace(':', '\\:');
```

Comment:
```suggestion
    const escapedUsername = OmegaUp.UI.escapeCss(OmegaUp.username);
```

y luego `OmegaUp.UI.escapeCss` puede ser

```javascript
escapeCss: function(s) {
  if (window.CSS && window.CSS.escape) return window.CSS.escape(s);
  return s.replace(':', '\\:');
},
```

o usa `CSS.escape` directo y agrega https://github.com/mathiasbynens/CSS.escape/blob/master/css.escape.js a los polyfills.

Reply:
sabes qué? ya lo pensé bien. esto está modificando el HTML rendereado de [`Scoreboard.vue`](https://github.com/omegaup/omegaup/blob/master/frontend/www/js/omegaup/components/arena/Scoreboard.vue)! entonces no deberíamos estarlo haciendo.

puedes mejor pasarle el nuevo ranking a `self.elements.rankingTable` para que Vue se encargue de actualizar el puntaje y toda la demás lógica de negocio que NO debería ir aquí?

Reply:
Listo